lib/extends/helpers/decidim/meetings/directory/application_helper_extends.rb

```diff
@@ -0,0 +1,31 @@
+# frozen_string_literal: true
+
+module Decidim
+  module Meetings
+    module Directory
+      module ApplicationHelperExtends
+        include Decidim::CheckBoxesTreeHelper
+        def scope_children_to_tree(scope)
+          return unless scope.children.any?
+
+          sorted_children = sort_children(scope.children)
+
+          sorted_children.flat_map do |child|
+            TreeNode.new(
+              TreePoint.new(child.id.to_s, translated_attribute(child.name, current_organization)),
+              scope_children_to_tree(child)
+            )
+          end
+        end
+
+        def sort_children(children)
+          children.sort_by { |child| child.name.values.first.split.first }
```

Comment:
Can you explain why you did that?

Reply:
Actually, I made it to make the code more readable because in my opinion it was better to target directly which function sorts if we need to change it in the future. Actually it was personal preference but I can change it if you prefer.

Reply:
No it's fine but I don't understant what it does

Reply:
We can probably sort on a method like `translated_attribute`